ERRORCODES.md

```diff
@@ -78,6 +78,7 @@ The following warnings are currently emitted by default:
 | Y062 | `Literal[]` slices shouldn't contain duplicates, e.g. `Literal[True, True]` is not allowed. | Redundant code
 | Y063 | Use [PEP 570 syntax](https://peps.python.org/pep-0570/) (e.g. `def foo(x: int, /) -> None: ...`) to denote positional-only arguments, rather than [the older Python 3.7-compatible syntax described in PEP 484](https://peps.python.org/pep-0484/#positional-only-arguments) (`def foo(__x: int) -> None: ...`, etc.). | Style
 | Y064 | Use simpler syntax to define final literal types. For example, use `x: Final = 42` instead of `x: Final[Literal[42]]`. | Style
+| Y065 | Don't use bare `Incomplete` in argument and return annotations. Instead, leave them unannotated. | Style
```

Comment:
Could you add a sentence about how both pyright and mypy treat unannotated parameters/returns differently to those explicitly annotated with `Incomplete`? (https://github.com/python/typeshed/issues/8472.) I think that's an important additional motivation for this rule.